scalardb-analytics-postgresql-sample/README.md

```diff
@@ -1,19 +1,40 @@
-# Sample Application for ScalarDB Analytics with PostgreSQL
+# Create a Sample Application That Uses ScalarDB Analytics with PostgreSQL
 
-This tutorial describes how to create a sample application for ScalarDB Analytics with PostgreSQL.
+This tutorial describes how to create a sample application that uses ScalarDB Analytics with PostgreSQL.
```

Comment:
I'm wondering whether "sample application" fits this example. This might be a matter of taste, but just "tutorial" is a bit more correct?

Reply:
Yeah, I understand what you mean and agree with it. I name this just to be consistent with other sample application documents. If we rename this, we move this to another place in the document site since it currently exists in the `Samples` category.

@josh-wong Do you have an idea where we should place this document?

Reply:
I see what you mean. We aren't really showing users how to set up a sample application in this case; rather, we're showing them how to run queries on sample data.

How about we do the following?:

- Change the title to `Run Queries on Sample Data by Using ScalarDB Analytics with PostgreSQL`?
- Change mentions of `the sample application` to some variation of `this tutorial`. (The exact wording will depend on the sentence.)

In this case, I think we can leave this document in the `Samples` category since this document still refers to setting up a sample scenario. 

Do you have any concerns about that?

Reply:
Sounds better to me!

> Run Queries on Sample Data by Using ScalarDB Analytics with PostgreSQL

The queries used in this sample are analytical queries. So, how about adding "Analytical" before Queries like `Run Analytical Queries on Sample Data by Using ScalarDB Analytics with PostgreSQL` ? But I don't have a strong opinion on it and it's okay without "Analytical".

Reply:
Thank you, guys! The title looks better! And yes, I also prefer to add "Analytical" to the title. It would be more informative for users.

Reply:
Got it, thank you! I've updated the title.